app/assets/stylesheets/application.bootstrap.scss

```diff
@@ -217,6 +217,12 @@ input.search-bar {
   &:focus {
     box-shadow: 0 0 0 0.25rem var(--brand-color-light);
   }
+
+  &:disabled {
+    color: gray !important;
+    pointer-events: none !important;
+    cursor: not-allowed !important;
+  }
```

Comment:
Theres a bootstrap `disabled` property that you can use instead https://react-bootstrap.github.io/components/buttons/#disabled-state

Reply:
It has the disabled property already but it is being overwritten by the custom button style.